site/docs/spark-queries.md

```diff
@@ -234,7 +260,7 @@ SELECT * FROM prod.db.table.manifests
 +----------------------------------------------------------------------+--------+-------------------+---------------------+------------------------+---------------------------+--------------------------+--------------------------------------+
 ```
 
-Note: 
+!!! Note
```

Comment:
Doesn't this create an empty note?

Reply:
yeah, I will resole it like this:
![image](https://user-images.githubusercontent.com/20125927/140733693-fc26e3a0-8e70-443e-9c0f-8959ea005963.png)

Reply:
I don't think that we need a callout box just because there were a couple of notes before.

Reply:
I don't think this should be in a note callout block. The previous formatting was just a paragraph, "Note:" with a list is better.

Reply:
@KnightChess, can you fix this?